_events/sr2020/southampton-kickstart.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Southampton Kickstart
+date: 2019-10-26 10:00:00
+layout: event
+type: kickstart
+location: Southampton University
+---
+
+Kickstart is the event which kicks off a competition year, where the game will
+be announced and robotics kits will be handed out to the teams.
+
+If you are unable to attend Kickstart, please [let us know][teams-contact] so we
+can arrange another way to get you your kit.
+
+We'll have desktops available for students to use, however feel free to bring personal laptops if you prefer.
+
+## Directions
+
+The Southampton kickstart is being held at the University of Southampton at:
+
+**TODO**
+
+For public transport or driving instructions, see the University of Southampton's information on [getting to our campuses][soton-campus-directions] (scroll down to Highfield campus).
+
+For parking and walking directions once you arrive, see the [on-campus directions][soton-directions].
+
+## Food
+
+There are plenty of places to get food on and nearby campus see [where to find food][soton-food-map].
+
+## Schedule
+
+| Time  | Info |
+|-------|------|
+| 10:00 | Introductory presentation, where the game of SR2020 will be announced. |
+| 11:00 | Team kits will be handed out, and teams will take part in activities to get them acquainted with their kits. |
+| 12:30 | There will be a lunch break. Teams are expected to bring their own lunches or get them from food vendors near the venues. |
+| 17:00 | Finish. |
+
+[teams-contact]: mailto:teams@studentrobotics.org
+[soton-food-map]: https://drive.google.com/open?id=1vDqr-yQh-OO7MzVa_OiuKD5xUrEsLprr
+[soton-directions]: {{ site.baseurl }}/resources/sr2019/southampton-kickstart-directions.pdf
```

Comment:
This likely needs updating for this year; the PDF has "SR2019" baked in.

Reply:
Quite likely. It's in the description to be updated once we actually know where kickstart will be.